src/pt/sussyscan/src/eu/kanade/tachiyomi/extension/pt/sussyscan/SussyToons.kt

```diff
@@ -331,9 +390,24 @@ class SussyToons : HttpSource() {
     private fun String.toDate() =
         try { dateFormat.parse(this)!!.time } catch (_: Exception) { 0L }
 
+    /**
+     * Normalizes path segments:
+     * Ex: [ "/a/b/", "/a/b", "a/b/", "a/b" ]
+     * Result: "a/b"
+     */
+    private fun String.toPathSegment() = this.trim().split("/")
+        .filter(String::isNotEmpty)
+        .joinToString("/")
+
     companion object {
-        const val CDN_URL = "https://usc1.contabostorage.com/23b45111d96c42c18a678c1d8cba7123:cdn"
+        const val CDN_URL = "https://cdn.sussytoons.site"
         const val OLDI_URL = "https://oldi.sussytoons.site"
+        const val mangaPagePrefix = "mangaPage:"
+        const val chapterPagePrefix = "chapterPage:"
+        const val pageImagePrefix = "pageImage:"
+
+        val chapterUrlRegex = """push\("(?<chapterUrl>[^"]*capitulo[^"]*)/?"\.concat""".toRegex()
+        val pageUrlRegex = """\.get\("(?<pageUrl>[^"]*capitulo[^(/?")]*)/?"\.concat""".toRegex()
```

Comment:
Avoid using named capture groups, they are not supported by the current minSDK (21).